src/plugins/intel_gpu/src/plugin/infer_request.cpp

```diff
@@ -239,7 +239,7 @@ void InferRequest::SetBlob(const std::string& name, const Blob::Ptr& data) {
                                            desc.getPrecision().size(),
                                            std::multiplies<size_t>());
 
-    if (dataBinSize != netReqBinSize && !compoundBlobPassed) {
+    if (dataBinSize != netReqBinSize && !compoundBlobPassed && !preProcessingRequired(foundInput, data)) {
```

Comment:
Is it about resize?

Reply:
Yes, in some cases the input blob is resized in the preproc and initially doesn't match the network